### PR TITLE
Add pycoast recipe

### PR DESCRIPTION
**Pull request by djhoese**
Another satpy dependency (and the last one I think). See #5883 for more details.

### CODE REVIEW

File: recipes/pycoast/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - aggdraw
+
+test:
+  imports:
+    - pycoast
+    - pycoast.tests
+
+about:
+  home: https://github.com/pytroll/pycoast
+  license: GPLv3
```
Comment (synapticarbors) [inline recipes/pycoast/meta.yaml]: `license: GPL-3.0`

https://opensource.org/licenses/alphabetical

File: recipes/pycoast/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/pycoast/meta.yaml]: Are any of the dependencies outside of `python`, `pip` and `setuptools` needed to build the package? A quick glance at the source code in `setup.py` looks like the others might not be needed in the `host` section.

File: recipes/pycoast/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  license: GPLv3
+  license_family: GPL3
+  license_file: LICENSE.txt
+  summary: Writing of coastlines, borders and rivers to images in Python
+  description: Writing of coastlines, borders and rivers to images in Python
+  doc_url: http://pycoast.readthedocs.io
+  dev_url: https://github.com/pytroll/pycoast
+
+extra:
+  recipe-maintainers:
```
Comment (synapticarbors) [inline recipes/pycoast/meta.yaml]: Can you confirm that all of the listed co-maintainers have agreed to be included?

File: recipes/pycoast/meta.yaml (hunk 1 above)
Comment (djhoese) [reply recipes/pycoast/meta.yaml]: @synapticarbors This naming was taken as part of the output from `conda skeleton`. Similarly the `host` requirements were from that too, I'll fix both I guess.

File: recipes/pycoast/meta.yaml (hunk 3 above)
Comment (djhoese) [reply recipes/pycoast/meta.yaml]: It was mentioned in another of the satpy dependency PRs that we are waiting on explicit permission from these co-maintainers. @adybbroe @pnuu @mraspaud

Comment (mraspaud) [reply recipes/pycoast/meta.yaml]: Yes, I answered that I was ok with being a maintainer.

Comment (pnuu) [reply recipes/pycoast/meta.yaml]: Same here, I'm ok with being a maintainer.

File: recipes/pycoast/meta.yaml (hunk 1 above)
Comment (djhoese) [reply recipes/pycoast/meta.yaml]: @synapticarbors FYI I was adding a license file to a different project and found this page for GPL talking about specifying future license versions: https://www.gnu.org/licenses/identify-licenses-clearly.html

Not sure if this affects the type of metadata you brought up here, but wanted to mention it.

**Pull request merged**